insights/parsers/satellite_postgresql_query.py

```diff
@@ -203,3 +205,34 @@ class SatelliteComputeResources(SatellitePostgreSQLQuery):
         'test_compute_resource1'
     """
     pass
+
+
+@parser(Specs.satellite_sca_status)
+class SatelliteSCAStatus(SatellitePostgreSQLQuery):
+    """
+    Parse the output of the command ``psql -d candlepin -c "select displayname, content_access_mode from cp_owner" --csv``.
+
+    .. note::
+        Please refer to its super-class :class:`insights.parsers.satellite_postgresql_query.SatellitePostgreSQLQuery` for more
+        details.
+
+    Sample output::
+
+        displayname,content_access_mode
+        Default Organization,entitlement
+        Orgq,org_environment
+
+    Examples:
+        >>> type(sat_sca_info)
+        <class 'insights.parsers.satellite_postgresql_query.SatelliteSCAStatus'>
+        >>> sat_sca_info.sca_enabled
+        True
+    """
+
+    @property
+    def sca_enabled(self):
```

Comment:
Since it's said `If the value of content_access_mode is "org_environment", it means the SCA is enabled for this organization`. 
I think it should be an `attribute` of the `"Organization"` returned by the SQL.

Or, it should be `If one of the organizations is SCA enabled, the Satellite is SCA enabled`, right

Reply:
@xiangce, yes, the latter one is right. Let me update the comment to make it more clear.